.github/workflows/pre-merge.yaml

```diff
@@ -16,6 +16,9 @@ jobs:
       fail-fast: false
     runs-on: ${{ matrix.os }}
 
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+
```

Comment:
because of https://github.com/getsentry/sentry-cli/issues/986
we already have this in the other GH workflows, this one was missing since we have new tests now